src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeWriter.cs

```diff
@@ -121,7 +121,7 @@ public static void AddPackage(string packageName, string packageVersion, string
                     arguments.Add(tfm);
                 }
 
-                consoleLogger.LogMessage($"\nAdding package {packageName} . . . ");
+                consoleLogger.LogMessage($"Adding package {packageName} . . . ");
```

Comment:
>$"Adding package [](http://example.com/codeflow?start=41&length=16)

english only for now?

Reply:
Added resources. Localization will be a later item to tackle. Tracking - #1595